centreon/src/CentreonNotification/Domain/Repository/EscalationRepository.php

```diff
@@ -48,33 +61,43 @@ public static function entityClass(): string
      */
     public function checkListOfIds(array $ids): bool
     {
-        return $this->checkListOfIdsTrait($ids);
+        return $this->checkListOfIdsTrait(
+            $ids,
+            Escalation::TABLE,
+            Escalation::ENTITY_IDENTIFICATOR_COLUMN
+        );
     }
 
     /**
      * {@inheritdoc}
      */
     public function getPaginationList($filters = null, int $limit = null, int $offset = null, $ordering = []): array
     {
-        $sql = 'SELECT SQL_CALC_FOUND_ROWS `esc_id`, `esc_name` '
-            . 'FROM `' . $this->getClassMetadata()->getTableName() . '`';
+        $sql = 'SELECT SQL_CALC_FOUND_ROWS `esc_id`, `esc_name` FROM `:db`.escalation';
 
         $collector = new StatementCollector();
 
         $isWhere = false;
         if ($filters !== null) {
-            if (array_key_exists('search', $filters) && $filters['search']) {
+            if (
+                array_key_exists('search', $filters)
+                && $filters['search']
+            ) {
```

Comment:
Consistency with other modifications of this type.

```suggestion
            if ($filters['search'] ?? false) {
```